tutorials/physics/interpolation/physics_interpolation_quick_start_guide.rst

```diff
@@ -0,0 +1,14 @@
+.. _doc_physics_interpolation_quick_start_guide:
+
+Quick start guide
+=================
+
+- Turn on physics interpolation: :ref:`ProjectSettings.physics/common/physics_interpolation<class_ProjectSettings_property_physics/common/physics_interpolation>`
+- Make sure you move objects and run your game logic in ``_physics_process()``
+  rather than ``_process()``. This includes moving objects directly *and
+  indirectly* (by e.g. moving a parent, or using another mechanism to automatically
+  move nodes).
+- Be sure to call :ref:`Node.reset_physics_interpolation<class_Node_method_reset_physics_interpolation>`
+  on nodes *after* you first position or teleport them, to prevent "streaking".
+- Temporarily try setting :ref:`ProjectSettings.physics/common/physics_ticks_per_second<class_ProjectSettings_property_physics/common/physics_ticks_per_second>`
```

Comment:
See the other comment about project settings for suggestions on how to style these links